src/components/section/section.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import  "./section.css"
+
+function Section() {
+  return (
+    <div className="section">
+    <div>
+       <img src="images/icon1.png"></img>
+       <h2>Declarative</h2>
+       <p>React makes painless to create interactive UIs.</p>
+       </div>
```

Comment:
Nicely done! As a bonus, you can try to refactor these divs into a component and render them here instead of repeating the structure over & over 👍